torch/csrc/jit/fusion_compiler.cpp

```diff
@@ -67,6 +149,17 @@ std::ostream& operator<<(std::ostream & out, const TensorDesc & d) {
   return out;
 }
 
+Node* usedInFusedChunk(Value * input) {
+  auto uses = input->uses();
+  if (uses.size() == 1) {
+    Node *user = uses[0].user;
+    if (user->kind() == prim::FusedChunk) {
```

Comment:
What happened to the usage assert?

Reply:
Writing out this code to do what it's supposed to do, *and* check the invariant would be annoying, because it would need a loop over uses. Since it's really an invariant, I don't think we need to assert it at every single step. It would be a loud failure if we were to violate it I guess.